### ISSUE TITLE
'Failed to parse JSON' when running script

### ISSUE THREAD

**Issue by peckmc** [G33kDude/Chrome.ahk#39]
I'm trying to upgrade from v1.2 to v1.3. When I include Chrome.ahk in a script I get this error:
```
Failed to parse JSON (-1,0)
Specifically: Unexpected character at position 0: "
```
v1.2 was working as expected 🤷 
Any help is much appreciated!

**Comment by G33kDude** [comment]
Does it give any indication of what code path triggered this error?

Are you using AutoHotkey Unicode?

**Comment by peckmc** [comment]
Thanks for your reply! The problem went away when I upgraded to AHK v2. I was definitely using the ANSI version before. Is it known to cause issues?

**Comment by G33kDude** [comment]
Yes, the new high performance JSON library is not compatible with ANSI AHK.
